components/Header.tsx

```diff
@@ -3,7 +3,10 @@ import Link from 'next/link';
 
 export default function Header({ coordinatorName }) {
   return (
+    <>
+    <title> {`Vortex | ${coordinatorName}`} </title>
     <div>
+      <link rel="shortcut icon" href="/favicon.ico" />
```

Comment:
Can move this line to the `<Head>` section [here](https://github.com/ln-vortex/ln-vortex-gui/blob/ccc23e0a7198bbebeb677a9222b0491e8fe9bedf/pages/_document.tsx#L12-L20)